common/configuration.go

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2024 Meroxa, Inc. & Yalantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"strings"
+)
+
+const (
+	MaxConfigStringLength = 127
+	ConfigTable           = "table"
+	ConfigKeyColumns      = "keyColumns"
+)
+
+// Configuration contains common for source and destination configurable values.
+type Configuration struct {
+	// DSN is the configuration of the data source name to connect to the Amazon Redshift.
+	DSN string `json:"dsn" validate:"required"`
+	// Table is the configuration of the table name.
+	Table string `json:"table" validate:"required"`
+	// KeyColumns is the configuration list of column names to build the opencdc.Record.Key (for Source).
+	KeyColumns []string `json:"keyColumns"`
```

Comment:
is this param only used by the source? if so, it should be moved under the source config not common

Reply:
It is used by destination also.

Reply:
can you update the comment on top of the param plz? it sounds like it's only for the source, what do we mean by `(for Source)`?

Reply:
@parikshitg also please check the tests since they're failing, and lemme know when you need another review

Reply:
Sure, @grvsahil looked into Localstack, I think there is some issues with redshift running in it. We look again.

Reply:
Adding a link to slack conversation. https://meroxa.slack.com/archives/C01ERTL0MJR/p1729163035429829?thread_ts=1728654070.170559&cid=C01ERTL0MJR